### ISSUE TITLE
Add ability to configure the Jira issue status upon creation

### ISSUE THREAD

**Issue by robertjahn** [keptn-sandbox/jira-service#17]
This enhancement will allow for the ability to define the status default for the service.  Currently, new Jira Issues from problems and evaluations are added to the backlog as either as a "backlog" for Kanban or "to do" for sprints.   Perhaps this goes in the jira details secret as:

```
    --from-literal="jira-default-problem-status=In Progress"
    --from-literal="jira-default-evaluation-status=Done"
```

**Comment by ghost** [comment]
Hi Rob, the intention here is that "open" tickets are tickets yet to be evaluated by a human.

What I'd suggest here is to:
1) Create a custom issue type dedicated to Keptn tickets.
2) Modify the `jira-details` secret flag to reference the name of this new issue type.

Once this is done, all keptn-derived tickets will be created under your custom issue type.

3) Modify the filter for your backlog / kanban board to only show issues of particular types: eg. everything _except_ your keptn tickets.

The same mechanism is usually employed to exclude Epics and Story tasks from the "work" backlog.

**Comment by robertjahn** [comment]
Yes I have the custom type now, but if I exclude them from the board then its not easy to find them since they get auto-generated.  Problems we do want to see on the board and the status I think would be "To do" for that one.  the evaluation from a quality gate for performance test that makes is evaluation task that yes, some one might want to review especially if it is a FAIL.  That is why I was thinking of letting this be configurable.... but we can get input from others on this one.

Here is one mockup.  If I add a state called "To review" and I could set the evaluation Jira issues to that, then the board would have a nice list. 

![image](https://user-images.githubusercontent.com/24907089/80440831-9ce7bb00-88d7-11ea-831e-27e4a851e1d6.png)
